Update nf-processthreadsapi-openprocesstoken.md

oblivioncth opened the pull request:
This page incorrectly states that the `OpenProcessToken` function
requires the provided HANDLE argument (ProcessHandle) to have the
`PROCESS_QUERY_INFORMATION` access permission, when it actually
only needs the `PROCESS_QUERY_LIMITED_INFORMATION` access permission.

This change corrects the statement.

I was informed of this discrepancy via StackOverflow and from my admittedly brief testing it appears to hold true.

**Additional Info**
This is the stackoverflow thread I am referring to: [Check if a different process is running with elevated privileges](https://stackoverflow.com/questions/73303801/check-if-a-different-process-is-running-with-elevated-privileges?noredirect=1#comment129468990_73303801)

This is roughly the checked I performed to confirm this change is accurate:

```
#include <iostream>
#include <windows.h>

DWORD procId = getProcessIdByName("notepad.exe"); // Custom function, uses CreateToolhelp32Snapshot method
HANDLE hProcess = OpenProcess(PROCESS_QUERY_LIMITED_INFORMATION, FALSE, procId);
HANDLE tProcess;
bool tokenOpened = OpenProcessToken(hProcess, TOKEN_QUERY, &tProcess);
std::cout << tokenOpened;
// Output: true
```

I have also successfully used the resultant token in functions like `GetTokenInformation` (to check `TokenElevation` as mentioned in the SO thread) without issue.

oblivioncth committed:
Update nf-processthreadsapi-openprocesstoken.md

This page incorrectly states that the `OpenProcessToken` function
requires the provided HANDLE argument (ProcessHandle) to have the
`PROCESS_QUERY_INFORMATION` access permission, when it actually
only needs the `PROCESS_QUERY_LIMITED_INFORMATION` access permission.

This change corrects the statement.

PRMerger14 commented:
@oblivioncth : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft commented:
Hi @oblivioncth - can you provide the link to the stackoverflow thread?

tunsira reviewed (approved):
Repository

oblivioncth commented:
@Karl-Bridge-Microsoft See the updated content of my original comment

Karl-Bridge-Microsoft committed:
Update nf-processthreadsapi-openprocesstoken.md

cleaned up some html/md formatting, added more detail that was included in the SO thread.

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/sdk-api?pullRequest=1268) <br/>All CLA requirements met.
